build.sbt

```diff
@@ -93,8 +92,7 @@ lazy val root = (project in file("."))
     publish / skip := true,
     publishLocal := {},
     publishM2 := {},
-    publishArtifact := false,
-    scapegoatDisabledInspections := Seq("AsInstanceOf", "BooleanParameter", "ComparingUnrelatedTypes", "FinalModifierOnCaseClass", "MethodNames")
+    publishArtifact := false
```

Comment:
In future can you please keep the comma at the end? sbt / Scala can handle that, and it's nicer for diffs if that line would not be modified.